src/components/Message/Message.tsx

```diff
@@ -44,7 +44,7 @@ export const Message = ({ activity }: MessageProps): JSX.Element => {
   return (
     <MessageViewer
       content={message.body}
-      subject={message.subject}
+      subject={message.subject!}
```

Comment:
@mohsinht do you think it is ok to go with this? Message.subject can be string or undefined it looks - something changed on graphql. We can either go with this approach (not very safe) or put subject as empty when message.subject is undefined. Wdyt?